### PR TITLE
initial commit of full view functionality

### PR DESCRIPTION
**Pull request by dwhitena**
- expanded view functionality for items along all sub paths.

### CODE REVIEW

**Comment by dwhitena** [comment]
I think there is a small conflict in the test fixtures.  Should be fixed before merging.

File: internal/wire/wire.go [hunk 1]
```diff
@@ -164,6 +164,7 @@ func viewPathToGraphPath(v *view.View, key string, graphDB *cayley.Handle) (*pat
 
 	// Loop over the path segments translating the path.
 	var graphPath *path.Path
+	var subPaths []path.Path
 	level := 1
 	for _, segment := range v.Path {
```
Comment (ardan-bkennedy) [inline internal/wire/wire.go]: This is silly stuff. These variable declarations should be separated from the for loop. They are really two separate concerns in the code.

Comment (ardan-bkennedy) [inline internal/wire/wire.go]: I saw this as well in one other space. Just to stay consistent.

Comment (ardan-bkennedy) [inline internal/wire/wire.go]: On line 125

**Pull request merged**